src/site/content/en/secure/fixing-mixed-content/index.md

```diff
@@ -205,33 +207,3 @@ attempting to upgrade requests that the browser currently does not.
 
 The `upgrade-insecure-requests` directive cascades into `<iframe>` documents,
 ensuring the entire page is protected.
-
```

Comment:
I've removed this section because it's [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/block-all-mixed-content). According to this doc mixed content has been completely blocked so it's obsolete. I'm not sure what sort of deprecation notice if any you'd like to attach here.

Reply:
I think it's fine to remove.